lib/go/thrift/simple_server.go

```diff
@@ -183,9 +186,6 @@ func (p *TSimpleServer) processRequests(client TTransport) error {
 			log.Printf("error processing request: %s", err)
 			return err
 		}
-		if err, ok := err.(TApplicationException); ok && err.TypeId() == UNKNOWN_METHOD {
-			continue
-		}
```

Comment:
And again I dunno why one of builds has failed, but it looks like it's nothing to do with Fo code.